### PR TITLE
Merge master

### PR DESCRIPTION
**Pull request by iphydf**

### CODE REVIEW

File: toxcore/net_crypto.c [hunk 1]
```diff
@@ -2537,16 +2537,18 @@ static void send_crypto_packets(Net_Crypto *c)
 
                 unsigned int pos = conn->last_sendqueue_counter % CONGESTION_QUEUE_ARRAY_SIZE;
                 conn->last_sendqueue_size[pos] = num_packets_array(&conn->send_array);
-                ++conn->last_sendqueue_counter;
 
                 long signed int sum = 0;
-                sum = (long signed int)conn->last_sendqueue_size[(pos) % CONGESTION_QUEUE_ARRAY_SIZE] -
-                      (long signed int)conn->last_sendqueue_size[(pos - (CONGESTION_QUEUE_ARRAY_SIZE - 1)) % CONGESTION_QUEUE_ARRAY_SIZE];
+                sum = (long signed int)conn->last_sendqueue_size[pos] -
```
Comment (zoff99) [inline toxcore/net_crypto.c]: @iphydf what will this do?

File: toxcore/net_crypto.c [hunk 2]
```diff
@@ -2537,16 +2537,18 @@ static void send_crypto_packets(Net_Crypto *c)
...
+                      (long signed int)conn->last_sendqueue_size[(pos + 1) % CONGESTION_QUEUE_ARRAY_SIZE];
 
                 unsigned int n_p_pos = conn->last_sendqueue_counter % CONGESTION_LAST_SENT_ARRAY_SIZE;
                 conn->last_num_packets_sent[n_p_pos] = packets_sent;
                 conn->last_num_packets_resent[n_p_pos] = packets_resent;
 
+                conn->last_sendqueue_counter = (conn->last_sendqueue_counter + 1) %
```
Comment (zoff99) [inline toxcore/net_crypto.c]: @iphydf what will this do?

Comment (iphydf) [reply toxcore/net_crypto.c]: https://github.com/TokTok/c-toxcore/pull/1268

File: toxcore/net_crypto.c (hunk 1 above)
Comment (iphydf) [reply toxcore/net_crypto.c]: https://github.com/TokTok/c-toxcore/pull/1268

**Pull request merged**